packages/expo-doctor/src/api/getSchemaAsync.ts

```diff
@@ -0,0 +1,9 @@
+import fetch from 'node-fetch';
+
+export async function getSchemaAsync(sdkVersion: string): Promise<any> {
+  const result = await fetch(
+    `https://exp.host/--/api/v2/project/configuration/schema/${sdkVersion}`
+  );
+  const resultJson = await result.json();
+  return resultJson.data.schema;
```

Comment:
This will crash if sdkVersion is not valid, endpoint output:
```json
{
"errors": [
{
"code": "VALIDATION_ERROR",
"message": "Version '44' is invalid. A version must be a valid semver or 'UNVERSIONED'.",
"isTransient": false
}
]
}
```

Reply:
cleaned it up a little... an invalid sdkVersion in app.json should fail before this in both old doctor and new